examples/python-guide/plot_example.py

```diff
@@ -37,7 +37,7 @@
                 lgb_train,
                 num_boost_round=100,
                 valid_sets=[lgb_train, lgb_test],
-                feature_name=['f' + str(i + 1) for i in range(X_train.shape[-1])],
+                feature_name=[f'f{i+1}' for i in range(X_train.shape[-1])],
```

Comment:
Unfortunately, our linting tool doesn't catch anything in f-string.
I think we should have whitespaces around arithmetic operator.
```suggestion
                feature_name=[f'f{i + 1}' for i in range(X_train.shape[-1])],
```